web/client/components/data/identify/DefaultViewer.jsx

```diff
@@ -95,6 +95,22 @@ class DefaultViewer extends React.Component {
         return validator.getValidResponses([response]);
     }
 
+    getInvalidTitles = () => {
+        const {invalidResponses} = this.getResponseProperties();
+        const titlesWithResponse = invalidResponses
+            .filter(({response}) => response !== "")
+            .map((res) => {
+                const {layerMetadata} = res;
+                return layerMetadata.title;
+            });
+        const titlesEmptyResponse = invalidResponses
+            .filter(({response}) => response === "")
+            .map((res) => {
+                const {layerMetadata} = res;
+                return layerMetadata.title;
+            });
+        return {titlesWithResponse, titlesEmptyResponse};
+    };
```

Comment:
I suggest to create a `getAlerts` that returns  an array of 
```j
{
  type: "info" // or "danger"
  msgId
  msgParams // layers
}
```

I think you can use HTML in any case.
It is easier to parse and handle errors of any kind this way. 

```javascript
<Alert bsStyle={type}>
  <HTML msgId={msgId} msgParams={msgParams}/>
</Alert>
```